static/src/javascripts/projects/common/modules/onward/tech-feedback.js

```diff
@@ -116,6 +117,10 @@ define([
      * override those at the time the email is sent.
      */
     return function () {
+        detect.adblockInUse.then(function(adblockInUse){
+            adblockBeingUsed = adblockInUse;
```

Comment:
Thank you @regiskuckaertz for explaining to me in person why it's not the same because of the event listeners :dancing_men: